feat: fcc-todo-app (#51367)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64fb285637fa1e1c222033e3.md

---
id: 64fb285637fa1e1c222033e3
title: Step 47
challengeType: 0
dashedName: step-47
---

# --description--

> Now add `formInputValuesUpdated` as the second compulsory condition of the `if` statement with the `AND` operator.

Reviewer: ```suggestion
Now add `formInputValuesUpdated` as the second mandatory condition in the `if` statement using the `AND` operator.
```

> This way, the `Cancel` and `Discard` buttons modal won't be shown to the user if they make no changes to the input fields while trying to edit a task.

Reviewer: ```suggestion
This way, the `Cancel` and `Discard` buttons in the modal won't be displayed to the user if they haven't made any changes to the input fields while attempting to edit a task.
```
